content/blog/2019-12-18-ml-based-phases-ordering-with-llvm/index.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "ML based phases ordering with LLVM"
+extra.author = "Qian Huang and Horace He"
+extra.bio = """
+Qian Huang is a junior undergraduate studying CS and Mathematics.
+
+Horace He is a senior undergraduate studying CS and Mathematics.
+"""
++++
+
+In this project, we continued to experiment with ML based phases ordering as in project 2, but using c++ programs and LLVM instead. Compared with project 2 using Bril, there are a lot more infrastructure supports for this project: more available optimization passes, actual hardware performance counters and benchmark programs. These allowed us to explore this topic more.
```

Comment:
- Please link to your Project 2 post so readers can get more context there.
- Capitalize C++.
- supports -> support